### DISCUSSION TITLE
My school email expires soon, does it disable my GitHub Education plan?

### DISCUSSION THREAD

**Discussion by Asayuki397** [community/community#49702] [GitHub Education] [answered]
Hi, I'm a high schooler in South Korea. And our school email expires when grades change. (In Korea, the new grade starts in March) As for the title, does it affect my GitHub education? (and Heroku student plan)

**Comment by bbkx226** [comment, marked as answer]
Hello! As far as I know, your school email address is used to verify your student status when you apply for the GitHub Education and Heroku student plans. However, if your school email address expires, you can still continue to use your GitHub account and Heroku account, but you may need to renew your student status verification by using an alternative method.

For example, GitHub Education provides alternative ways to verify your student status, such as uploading a document that shows your name and school information or connecting your GitHub account to your school's single sign-on system. You can find more information about verifying your student status on the GitHub Education website.

**Reply by Asayuki397** [reply]
Thanks for the fast answer! I'll try it right away.
